### PR TITLE
chore: #1 Project skeleton

### PR DESCRIPTION
**Pull request by llorllale**
PR for #1:

* README (with badges)
* Makefile
* Travis-CI integration
* Golangci-lint integration

Signed-off-by: George Aristy <george.aristy@securekey.com>

### CODE REVIEW

**Comment by troyronda** [comment]
verify

**Comment by llorllale** [comment]
@troyronda reopened PR to trigger the Travis build

File: Makefile [hunk 1]
```diff
@@ -0,0 +1,29 @@
+#
+# Copyright SecureKey Technologies Inc. All Rights Reserved.
+#
+# SPDX-License-Identifier: Apache-2.0
+#
+
+#
+# Supported Targets:
```
Comment (troyronda) [inline Makefile]: suggest using a common target words across the repos, when applicable.

Comment (troyronda) [reply Makefile]: FYI, fabric & fabric-sdk-go use:
- unit-test
- integration-test
- checks
- lint (fabric-sdk-go); linter (fabric). (so lint is okay).
- neither fabric-sdk-go nor fabric have an explicit "build" target. (SDK is library while fabric has built targets like "peer" & "orderer".

Comment (llorllale) [reply Makefile]: @troyronda doesn't it make sense to have a `build` target though? A user might want to just build the binary w/o running any checks

Comment (llorllale) [reply Makefile]: In the future this could output a docker image or something.

**Comment by troyronda** [comment]
Yes a build target makes sense.

File: Makefile (hunk 1 above)
Comment (troyronda) [reply Makefile]: Yes a build target makes sense.

File: README.md [hunk 2]
```diff
@@ -1 +1,37 @@
...
+# run integration tests
+make integration-tests
+```
+
+## Build dependencies
+
+* Go `1.12.x`
+* Docker `18.09.x` or above
+* [golangci-lint](https://github.com/golangci/golangci-lint) v1.16.0
+    * Download the binary manually or using their [script](https://install.goreleaser.com/github.com/golangci/golangci-lint.sh)
```
Comment (troyronda) [inline README.md]: I like using the gobin tool for these dependencies.

**Pull request merged**